packages/server/src/integrations/base/sql.ts

```diff
@@ -646,6 +646,30 @@ class SqlQueryBuilder extends SqlTableQueryBuilder {
     }
     return results.length ? results : [{ [operation.toLowerCase()]: true }]
   }
+
+  async getBulkValues(rowsToInsert: Array<any>, source?: SqlClient) {
+    let finalValues = ""
+    const escapeCharacter = source === SqlClient.POSTGRES ? "E" : ""
+    for (let row of rowsToInsert) {
+      let values = ""
+      for (let value of Object.values(row)) {
+        if (typeof value === "string") {
+          values += `${escapeCharacter}'${value.replace(
+            /[\/#+\-&|!(){}\]^"~'*?:\\]/g,
```

Comment:
This regex is a bit hard to understand, what is the purpose here?

Reply:
Is there anyway we could de-duplicate it in that case, at least create a named variable in `shared-core` or `backend-core` that it can be used by Lucene and SQL?